lib/mathlib.cpp

```diff
@@ -143,6 +143,8 @@ double MathLib::toDoubleNumber(const std::string &str)
     // nullcheck
     else if (isNullValue(str))
         return 0.0;
+    else if (isFloat(str)) // Workaround libc++ bug at http://llvm.org/bugs/show_bug.cgi?id=17782
```

Comment:
I do agree that it'd be better not to have this, however I propose that to avoid regularly submitting PRs that remove uses of cbegin() and cend() so that people using a Mac can contribute to the project.
I believe it's better to have a 2-line workaround than working around the issue everywhere people (rightly) want to use cbegin() and cend().